opennms-model/src/main/java/org/opennms/netmgt/model/topology/BroadcastDomain.java

```diff
@@ -504,7 +503,11 @@ public String printTopologyFromLevel(Set<Integer> bridgeIds, int level) {
         strbfr.append(bridgeIds);
         strbfr.append("\n");
         for (Integer bridgeid : bridgeIds) {
-        	strbfr.append(getBridge(bridgeid).printTopology());
+            final Bridge bridge = getBridge(bridgeid);
+            if (bridge == null) {
+                continue;
```

Comment:
Can we use the Java Stream API instead and filter for the bridges which are actually available instead of using continue here. Something like this:
```java
bridgeIds.stream().map(id -> getBridge(id)).filter(bridge -> bridge != null).forEach(bridge -> {

....

})
```

Reply:
Ok